src/Sfa.Tl.Matching.Web/Controllers/OpportunityController.cs

```diff
@@ -89,6 +89,9 @@ public async Task<IActionResult> CheckAnswersReferrals(CheckAnswersReferralViewM
             var dto = _mapper.Map<CheckAnswersDto>(viewModel);
             await _opportunityService.UpdateOpportunity(dto);
 
+            await _referralService.SendEmployerReferralEmail(dto.OpportunityId);
+            await _referralService.SendProviderReferralEmail(dto.OpportunityId);
```

Comment:
We could probably do these in parallel using Task.WhenAll as they aren't dependant on eachother

Reply:
Changed as suggested.